### ISSUE TITLE
Rust 2018 edition absolute crate paths not resolved

### ISSUE THREAD

**Issue by markrendle** [intellij-rust/intellij-rust#3119]
I couldn't tell if this was referenced in #2622 anywhere, because I don't know the Rust feature name, so sorry if this is a duplicate.

## Environment

* Plugin 2018-11-26
* 2018 Edition (1.31)
* CLion 2018.3.1
* Windows 10 October 2018 Update

## Problem description

Rust 2018 edition removes the need for `extern crate` and just uses `use` but for some crates (e.g. `actix`) the compiler makes you prefix the name with `::`, like `use ::actix::prelude::*`. But if you put that `::` prefix on, IntelliJ-Rust loses all its type info.

## Steps to reproduce

- Create app with `edition = "2018"` in `Cargo.toml`
- Add `actix`
- Add `use ::actix::prelude::*` to `main.rs`
- Try to get completion on any `actix` types

## Compiler error without the `::` prefix

```
error[E0432]: unresolved import `actix::prelude`                                                                             
 --> src/main.rs:1:12                                                                                                        
  |                                                                                                                          
1 | use actix::prelude::*;                                                                                                   
  |            ^^^^^^^ could not find `prelude` in `actix`                                                                   
                                                                                                                             
error[E0659]: `actix` is ambiguous (name vs any other name during import resolution)                                         
 --> src/main.rs:1:5                                                                                                         
  |                                                                                                                          
1 | use actix::prelude::*;                                                                                                   
  |     ^^^^^ ambiguous name                                                                                                 
  |                                                                                                                          
  = note: `actix` could refer to an extern crate passed with `--extern`                                                      
  = help: use `::actix` to refer to this extern crate unambiguously                                                          
note: `actix` could also refer to the module imported here                                                                   
 --> src/main.rs:1:5                                                                                                         
  |                                                                                                                          
1 | use actix::prelude::*;                                                                                                   
  |     ^^^^^^^^^^^^^^^^^                                                                                                    
  = help: use `crate::actix` to refer to this module unambiguously                                                           
                                                                                                                             
error: aborting due to 2 previous errors                   
```

**Comment by rvlzzr** [comment]
Until this is resolved you can do something like `use ::{actix::prelude::*};` to get completion to work at least.

**Comment by Undin** [comment]
Fixed in #3768
